### ISSUE TITLE
withdrawInvestment

### ISSUE THREAD

**Issue by lilnamcor** [dharmaprotocol/LoanStandard#1]
[https://github.com/dharmaprotocol/LoanStandard/blob/master/contracts/Loan.sol#L140](url)

If 1 person invests then immediately withdraws this'll cause the contract to destruct, right?

**Comment by nadavhollander** [comment]
Yes -- but only if the time lock period has passed and the loan is still not fully funded.

**Comment by lilnamcor** [comment]
Gotcha, quick response lol. I didn't get to time locks yet
